README.md

```diff
@@ -149,10 +149,32 @@ regex.
 
 ### Releasing
 
-On a new branch, run `npm version <major|minor|patch|...> -m 'rel: %s'` (see
-`npm help version` for details). It will update the version in `package.json`,
-commit the changes, and create a git tag. Now you push the branch, get it
-reviewed and merged.
+On a new branch, run
+
+```
+# tsickle releases are all minor releases for now, see npm help version.
+$ npm version minor -m 'rel: %s'
+```
+
+It will update the version in `package.json`, commit the changes, and
+create a git tag. Now you push the branch, get it reviewed and merged.
+
+The "release" tag will not be included with the branch.  Unfortunately
```

Comment:
Maybe the trick is to push from the command line in your client?

Reply:
PTAL